angular/src/lib/icon/icon.component.ts

```diff
@@ -8,10 +8,7 @@ import {
   ViewEncapsulation,
 } from '@angular/core';
 
-import chain from 'lodash-es/chain';
-import find from 'lodash-es/find';
-import trimEnd from 'lodash-es/trimEnd';
-import uniqueId from 'lodash-es/uniqueId';
+import { chain, find, trimEnd, uniqueId } from 'lodash';
```

Comment:
Per our tree-shaking, I think we want to keep these imports separate. @pauljeter correct?

Reply:
@bfbiggs why is that?

Reply:
If you import it using 
```
import { chain, find, trimEnd, uniqueId } from 'lodash';
```

you bundle will not just include those four utility functions, but will have all of the lodash's utility functions